### ISSUE TITLE
Accessibility: Usability: Narrator:Narrator fails to announce ">" symbol

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#8]
#A11yUsable
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
FireFox Version: 50.1.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/docs/runtimes/nodejs](https://code.visualstudio.com/docs/runtimes/nodejs) in EDGE
2. Navigate to the text "View > Integrated Terminal" under "Integrated Terminal" header.
3. Observe the narrator announcement

**Actual Result:**
The Narrator fails to announce the ">" symbol.

**_Inspect Behaviour:_**
Name: ">"
ControlType: UIA_TextControlTypeId (0xC364)
LocalizedControlType: "text"
HelpText: ""

**Expected Result:**
The Narrator should announce the name of the symbol.

**User Impact:**
Users who rely on screen readers will not be aware of the ">" symbol and will be missed out on the data completely.

**Screenshots:**
![narrator buddy screenshot](https://cloud.githubusercontent.com/assets/24825441/21674660/b5f57ff6-d354-11e6-97e6-7413a21ad864.png)
![narrator fails to announce the name](https://cloud.githubusercontent.com/assets/24825441/21674659/b5f4b8dc-d354-11e6-93f6-7c1a577d9097.png)

**Comment by chrisdias** [comment]
No longer exists in docs, now it says "View | Integrated Terminal".  the "|" is not announced and i actually think this might be the right thing to do as it says "Use the View Integrated Terminal menu command".
